test/test_pullparser.rb

```diff
@@ -63,20 +63,24 @@ def test_entity_replacement
     end
 
     def test_character_references
-      source = '<a>&#65;</a><b>&#x42;</b>'
+      source = "<a>&#65;</a>\r\n<b>&#x42;</b>"
```

Comment:
Could you create a new test for `\r\n` case?
It's not related to character reference.

Reply:
OK
I see.